src/components/Gallery/Gallery.tsx

```diff
@@ -1,24 +1,32 @@
-// @ts-check
-import React, { useContext, useMemo, useState } from 'react';
-import PropTypes from 'prop-types';
-import { TranslationContext } from '../../context';
-import ModalWrapper from './ModalWrapper';
+import React, { useMemo, useState } from 'react';
+
+import { ModalComponent as ModalWrapper } from './ModalWrapper';
+
+import { useTranslationContext } from '../../context/TranslationContext';
+
+import type { Attachment } from 'stream-chat';
+
+import type { DefaultAttachmentType, UnknownType } from '../../../types/types';
 
 /**
```

Comment:
i think this comment block in the wrong spot, needs to be above UnMemoizedGallery, also let's add a comment block for images since this has an .md file and is getting built to docs